src/Integration/Binding/BindingProcessImpl.cs

```diff
@@ -85,14 +85,6 @@ public bool PromptSaveSolutionIfDirty()
             return result;
         }
 
-        public bool DiscoverBindableProjects()
```

Comment:
I've removed:
* `DiscoverBindableProjects` from the interface `IBindingProcess`
* `DiscoverBindableProjects` from the class `BindingProcess`
* `DiscoverProjects` from `BindingWorkflow` which just called `IBindingProcess.DiscoverBindableProjects`